query.js

```diff
@@ -543,6 +543,9 @@ exports.projectForEntryList = projectForEntryList
 
 function parseExport(res) {
 
+    var mentions = [];
```

Comment:
@ryanctan how exactly did these latest changes fix the export?

Reply:
So I actually don't really know why the commented-out code didn't work, I kinda just combined that code with the code for activities and travels and it worked. Sorry I couldn't be more helpful!

Reply:
Hmm, code is way more complicated, but it works, so let's merge!